### ISSUE TITLE
Blink/ElectronDisplay

### ISSUE THREAD

**Issue by ghuba** [sglyon/PlotlyJS.jl#164]
Although the behavior is very inconsistent, in most cases I am unable to get plots from the REPL anymore. Can anyone confirm?

Julia Version 0.6.2
MacOS High Sierra
PlotlyJS 0.8.2
Blink 0.6.0

For example, in the REPL:
```
julia> using PlotlyJS
julia> x = linspace(0,2*pi,128)
julia> plot(x,cos.(x))
```
The Blink window appears and stays there with the black square. 

<img width="811" alt="screen shot 2018-01-11 at 8 40 39 am" src="https://user-images.githubusercontent.com/4511360/34835963-4996e252-f6ab-11e7-9481-54845e36489d.png">

The plot never shows up. It looks like nothing is being sent to the Blink window.

On occasions, I get a blank window with no square. A few times the plot comes up but it's not reproducible. Very confusing...

**Comment by sglyon** [comment]
Sorry for the inconvenience.

This is a known issue with Blink v0.6.0.

It has been fixed in PlotlyJS version 0.8.3: https://github.com/sglyon/PlotlyJS.jl/releases/tag/v0.8.3

The pull request in METADATA.jl is still pending merge: https://github.com/JuliaLang/METADATA.jl/pull/12828#issuecomment-356823817

Once that happens a `Pkg.update()` should fix the problem

Until then you can `Pkg.checkout("PlotlyJS")` to get a working version again

**Comment by ghuba** [comment]
Thanks. Followed your instructions and this fixed the issue.
